TERMSOFSERVICE.md

```diff
@@ -69,19 +66,64 @@ by means of an individual agreement.
 reasons. If a registration is incomplete, we are entitled to delete the user
 account within one week.
 
-# 4. Account, password and security
+# 4. Account, password and security, Requirements for Usage, Privacy, Sanctions and Blocking of personal Accounts
 
 4.1. Registering and using Tideways may involve you setting up an account
 and giving a password.
 
-4.2. You are fully responsible for maintaining the confidentiality of your
-account and the password, and all activities that occur under your account or
-password. You agree to immediately notify us of any unauthorized use of your
+4.2. Consumers may not register based on the preceding section 4.1.. A
+corporation entity (legal person or private company) may only be registered by
+its lawful representatives and under the submission of their names. Each user
+is obliged to give true and complete information in the context of registration
+resting upon section 4.1.. Each user shall inform Tideways about any change in
+the contract details or the other user information without delay and unasked.
+If asked those information have to be confirmed.
+
+4.3. You may register and create an account only once. Your account may not be
+transferred. You are fully responsible for maintaining the confidentiality of
+your account and the password, and all activities that occur under your account
+or password. You agree to immediately notify us of any unauthorized use of your
 account or password and any breach of security or misuse or suspected breach of
-security or misuse of Tideways, and ensure that you exit from your
-account at the end of each session if you use a shared computer. Tideways cannot
-and will not be liable for any loss or damage arising from your failure to
-comply with this clause.
+security or misuse of Tideways, and ensure that you exit from your account at
+the end of each session if you use a shared computer. Tideways cannot and will
+not be liable for any loss or damage arising from your failure to comply with
+this clause.
+
+4.4. In case of reasonable grounds that your credentials have been obtained by
```

Comment:
With this clause we retain the right to disable, change your passwords or log out your user account automatically should we get knowledge that it was accessed by an unauthorized third party.